portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,56 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <meta name="description" content="Kenneth's Personal Website">
+    <meta name="keywords" content="Kenneth, Kiprotich">
+    <meta name="author" content="Kenneth Kiprotich">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h2>About Me</h2>
+      <p> I am Kenneth Kiprotich, a student Columbia Univerity, studying Computer Science.
+          I am interested in drawing and I enjoy early morning runs. </p>
+      <div class="image-thumbs">
+          <a href="images/KennethKiprotich.jpg"><img src="images/KennethKiprotich.jpg"/></a>
+      </div>
+      <p>This is my portfolio.</p> 
       <p>Click here to get a random message:</p>
+     
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
+
+      <div class="interests">
+        <h2>Interests</h2>
+             <ul>
+              <li><i>Running </i></li>
+              <li><i>Cooking</i></li>   
+              <li><i>Movies</i></li>
+            </ul>
+      </div>  
+      <div class="work">
+        <h2> Work </h2>
+        <p>•  Summer 2019- Software Developer Intern at BRCK: This summer (2019), I worked at 
```

Comment:
I thought I commented on this but looks like not.
Use `<ul>` for listing and adding styles later.

Reply:
Changed it to `<ul>` tags